src/charm.py

```diff
@@ -76,6 +76,7 @@ def main(self, event):
                         "USERID_HEADER": "kubeflow-userid",
                         "USERID_PREFIX": "",
                         "SESSION_STORE_PATH": "bolt.db",
+                        "OIDC_STATE_STORE_PATH": "oidc_state.db",
```

Comment:
Could you please add a comment here saying why this is needed, you could link [this](https://github.com/arrikto/oidc-authservice/blob/58427f7e2a3070f31dc152a1d86eb415ddf59ad8/common/settings.go#L23) part of the code.